install_config/install/deploy_router.adoc

```diff
@@ -269,9 +279,89 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be used to distribute routing load in the cluster. Because
+each router uses ports 80 (http) and 443 (https) the routers are placed on nodes
+in the cluster that are not already using those ports.
+
+Each router handles a set of routes that are tagged using labels. Router sharding
+uses ROUTE_LABELS. A project's (namespace) routes can be associated with a router
+using NAMESPACE_LABELS.
+
+
+[[using-namespace-labels]]
+=== Using NAMESPACE_LABELS
+
+Each project has its own namespace. The routes for a project can be handled by a
```

Comment:
I think we reword this to talk about namespaces more than projects.  It's worth linking to https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/projects_and_users.html#projects and mentioning that a project will create a new namespace... but focus more on the namespaces than projects.